static/src/js/components/SpatialDisplay/SpatialDisplay.js

```diff
@@ -281,14 +446,7 @@ class SpatialDisplay extends Component {
           title="Point"
         />
       ))
-    } else if ((boundingBoxSearch && !drawingNewLayer) || drawingNewLayer === 'rectangle') {
-      // Arrange the points in the right order
-      const points = boundingBoxSearch
-        ? boundingBoxSearch
-          .match(/[^,]+,[^,]+/g)
-          .map(pointStr => trimSpatial(pointStr.split(','), 5).reverse().join(', '))
-        : [undefined, undefined]
-
+    } else if ((boundingBoxSearch && (boundingBoxSearch[0] || boundingBoxSearch[1]) && !drawingNewLayer) || drawingNewLayer === 'rectangle' || manuallyEntering) {
```

Comment:
These `if` statements are getting pretty crazy. Wonder if theres a way they could be shortened up.

Reply:
Probably true. Can we do that with the advanced search as that will add yet another layer to this.